xchange-binance/src/main/java/org/knowm/xchange/binance/BinanceExchange.java

```diff
@@ -77,6 +78,11 @@ public boolean isFuturesSandbox(){
             exchangeSpecification.getExchangeSpecificParametersItem(SPECIFIC_PARAM_USE_FUTURES_SANDBOX));
   }
 
+  public boolean isFuturesEnabled(){
+    return Boolean.TRUE.equals(
+            exchangeSpecification.getExchangeSpecificParametersItem(SPECIFIC_PARAM_FUTURES_ENABLED));
+  }
```

Comment:
wont this turn off futures for every one after the upgrade?

wont this backward incompatibility be painful for existing clients?

Reply:
support for futures on binance is just being introduced, the first commit was about three months ago. I don't think there's anything to break in terms of backwards compatibility...